dipy/workflows/multi_io.py

```diff
@@ -253,3 +253,10 @@ def __iter__(self):
         IO = np.concatenate([I, O], axis=1)
         for i_o in IO:
             yield i_o
+
+    @staticmethod
```

Comment:
Why you need it as a static method?  I think you can remove it

Reply:
Thanks for the suggestion. I have removed the @staticmethod.